### ISSUE TITLE
Agent crash due to panic

### ISSUE THREAD

**Issue by sayboras** [cilium/cilium#20903]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Cilium agent crashed with the below stacktrace after configured below network policy with named port. 

Restoration was done by recreate the same network policy, this could be a race condition somewhere.

```yaml
apiVersion: cilium.io/v2
kind: CiliumNetworkPolicy
metadata:
  namespace: cilium-test
  name: client-egress-to-echo-deny-named-port
spec:
  endpointSelector:
    matchLabels:
      kind: client
  egressDeny:
  - toPorts:
    - ports:
      - port: "http-8080"
        protocol: TCP
    toEndpoints:
    - matchLabels:
        io.kubernetes.pod.namespace: cilium-test
        kind: echo
```

### Cilium Version

1.12.0

### Kernel Version

$ uname -a                                
Linux fedora 5.18.16-200.fc36.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Aug 3 15:44:49 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

### Kubernetes Version

$ k version                                  
Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.6", GitCommit:"ad3338546da947756e8a88aa6822e9c11e7eac22", GitTreeState:"clean", BuildDate:"2022-04-14T08:49:13Z", GoVersion:"go1.17.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.14-eks-18ef993", GitCommit:"ac73613dfd25370c18cbbbc6bfc65449397b35c7", GitTreeState:"clean", BuildDate:"2022-07-06T18:06:50Z", GoVersion:"go1.16.15", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.23) and server (1.21) exceeds the supported minor version skew of +/-1

### Relevant log output

<details>
<summary>stacktrace</summary>

```shell
2022-08-15T10:45:17.285404151Z level=info msg="Started gops server" address="127.0.0.1:9890" subsys=daemon
2022-08-15T10:45:17.285653766Z level=info msg="Memory available for map entries (0.003% of 8036806656B): 20092016B" subsys=config
2022-08-15T10:45:17.285706598Z level=info msg="option bpf-ct-global-tcp-max set by dynamic sizing to 131072" subsys=config
2022-08-15T10:45:17.285712717Z level=info msg="option bpf-ct-global-any-max set by dynamic sizing to 65536" subsys=config
2022-08-15T10:45:17.285716224Z level=info msg="option bpf-nat-global-max set by dynamic sizing to 131072" subsys=config
2022-08-15T10:45:17.285719742Z level=info msg="option bpf-neigh-global-max set by dynamic sizing to 131072" subsys=config
2022-08-15T10:45:17.285722643Z level=info msg="option bpf-sock-rev-map-max set by dynamic sizing to 65536" subsys=config
2022-08-15T10:45:17.290871014Z level=info msg="  --agent-health-port='9879'" subsys=daemon
2022-08-15T10:45:17.290887098Z level=info msg="  --agent-labels=''" subsys=daemon
2022-08-15T10:45:17.290891513Z level=info msg="  --agent-not-ready-taint-key='node.cilium.io/agent-not-ready'" subsys=daemon
2022-08-15T10:45:17.290894822Z level=info msg="  --allocator-list-timeout='3m0s'" subsys=daemon
2022-08-15T10:45:17.290897924Z level=info msg="  --allow-icmp-frag-needed='true'" subsys=daemon
2022-08-15T10:45:17.290900928Z level=info msg="  --allow-localhost='auto'" subsys=daemon
2022-08-15T10:45:17.290903829Z level=info msg="  --annotate-k8s-node='false'" subsys=daemon
2022-08-15T10:45:17.290906718Z level=info msg="  --api-rate-limit=''" subsys=daemon
2022-08-15T10:45:17.290910145Z level=info msg="  --arping-refresh-period='30s'" subsys=daemon
2022-08-15T10:45:17.290913693Z level=info msg="  --auto-create-cilium-node-resource='true'" subsys=daemon
2022-08-15T10:45:17.290917291Z level=info msg="  --auto-direct-node-routes='false'" subsys=daemon
2022-08-15T10:45:17.290920426Z level=info msg="  --bgp-announce-lb-ip='false'" subsys=daemon
2022-08-15T10:45:17.290923792Z level=info msg="  --bgp-announce-pod-cidr='false'" subsys=daemon
2022-08-15T10:45:17.290927009Z level=info msg="  --bgp-config-path='/var/lib/cilium/bgp/config.yaml'" subsys=daemon
2022-08-15T10:45:17.290930191Z level=info msg="  --bpf-ct-global-any-max='262144'" subsys=daemon
2022-08-15T10:45:17.290933475Z level=info msg="  --bpf-ct-global-tcp-max='524288'" subsys=daemon
2022-08-15T10:45:17.290936828Z level=info msg="  --bpf-ct-timeout-regular-any='1m0s'" subsys=daemon
2022-08-15T10:45:17.290940184Z level=info msg="  --bpf-ct-timeout-regular-tcp='6h0m0s'" subsys=daemon
2022-08-15T10:45:17.290943818Z level=info msg="  --bpf-ct-timeout-regular-tcp-fin='10s'" subsys=daemon
2022-08-15T10:45:17.290947692Z level=info msg="  --bpf-ct-timeout-regular-tcp-syn='1m0s'" subsys=daemon
2022-08-15T10:45:17.290951326Z level=info msg="  --bpf-ct-timeout-service-any='1m0s'" subsys=daemon
2022-08-15T10:45:17.290954871Z level=info msg="  --bpf-ct-timeout-service-tcp='6h0m0s'" subsys=daemon
2022-08-15T10:45:17.290965546Z level=info msg="  --bpf-ct-timeout-service-tcp-grace='1m0s'" subsys=daemon
2022-08-15T10:45:17.290969175Z level=info msg="  --bpf-filter-priority='1'" subsys=daemon
2022-08-15T10:45:17.290972634Z level=info msg="  --bpf-fragments-map-max='8192'" subsys=daemon
2022-08-15T10:45:17.290976100Z level=info msg="  --bpf-lb-acceleration='disabled'" subsys=daemon
2022-08-15T10:45:17.290979796Z level=info msg="  --bpf-lb-affinity-map-max='0'" subsys=daemon
2022-08-15T10:45:17.290983134Z level=info msg="  --bpf-lb-algorithm='random'" subsys=daemon
2022-08-15T10:45:17.290986567Z level=info msg="  --bpf-lb-dev-ip-addr-inherit=''" subsys=daemon
2022-08-15T10:45:17.290989886Z level=info msg="  --bpf-lb-dsr-dispatch='opt'" subsys=daemon
2022-08-15T10:45:17.290993756Z level=info msg="  --bpf-lb-dsr-l4-xlate='frontend'" subsys=daemon
2022-08-15T10:45:17.290997106Z level=info msg="  --bpf-lb-external-clusterip='false'" subsys=daemon
2022-08-15T10:45:17.291000469Z level=info msg="  --bpf-lb-maglev-hash-seed='JLfvgnHc2kaSUFaI'" subsys=daemon
2022-08-15T10:45:17.291003736Z level=info msg="  --bpf-lb-maglev-map-max='0'" subsys=daemon
2022-08-15T10:45:17.291006698Z level=info msg="  --bpf-lb-maglev-table-size='16381'" subsys=daemon
2022-08-15T10:45:17.291009628Z level=info msg="  --bpf-lb-map-max='65536'" subsys=daemon
2022-08-15T10:45:17.291012673Z level=info msg="  --bpf-lb-mode='snat'" subsys=daemon
2022-08-15T10:45:17.291015578Z level=info msg="  --bpf-lb-rev-nat-map-max='0'" subsys=daemon
2022-08-15T10:45:17.291018581Z level=info msg="  --bpf-lb-rss-ipv4-src-cidr=''" subsys=daemon
2022-08-15T10:45:17.291021150Z level=info msg="  --bpf-lb-rss-ipv6-src-cidr=''" subsys=daemon
2022-08-15T10:45:17.291023872Z level=info msg="  --bpf-lb-service-backend-map-max='0'" subsys=daemon
2022-08-15T10:45:17.291026685Z level=info msg="  --bpf-lb-service-map-max='0'" subsys=daemon
2022-08-15T10:45:17.291029543Z level=info msg="  --bpf-lb-sock='false'" subsys=daemon
2022-08-15T10:45:17.291032382Z level=info msg="  --bpf-lb-sock-hostns-only='false'" subsys=daemon
2022-08-15T10:45:17.291035170Z level=info msg="  --bpf-lb-source-range-map-max='0'" subsys=daemon
2022-08-15T10:45:17.291038212Z level=info msg="  --bpf-map-dynamic-size-ratio='0.0025'" subsys=daemon
2022-08-15T10:45:17.291041487Z level=info msg="  --bpf-nat-global-max='524288'" subsys=daemon
2022-08-15T10:45:17.291044222Z level=info msg="  --bpf-neigh-global-max='524288'" subsys=daemon
2022-08-15T10:45:17.291047232Z level=info msg="  --bpf-policy-map-max='16384'" subsys=daemon
2022-08-15T10:45:17.291050470Z level=info msg="  --bpf-root='/sys/fs/bpf'" subsys=daemon
2022-08-15T10:45:17.291053469Z level=info msg="  --bpf-sock-rev-map-max='262144'" subsys=daemon
2022-08-15T10:45:17.291056383Z level=info msg="  --bypass-ip-availability-upon-restore='false'" subsys=daemon
2022-08-15T10:45:17.291062925Z level=info msg="  --certificates-directory='/var/run/cilium/certs'" subsys=daemon
2022-08-15T10:45:17.291065823Z level=info msg="  --cflags=''" subsys=daemon
2022-08-15T10:45:17.291067906Z level=info msg="  --cgroup-root='/run/cilium/cgroupv2'" subsys=daemon
2022-08-15T10:45:17.291069849Z level=info msg="  --cluster-health-port='4240'" subsys=daemon
2022-08-15T10:45:17.291071747Z level=info msg="  --cluster-id='0'" subsys=daemon
2022-08-15T10:45:17.291073628Z level=info msg="  --cluster-name='tammach-ap-southeast-2-eksctl-io'" subsys=daemon
2022-08-15T10:45:17.291075629Z level=info msg="  --clustermesh-config='/var/lib/cilium/clustermesh/'" subsys=daemon
2022-08-15T10:45:17.291077548Z level=info msg="  --cmdref=''" subsys=daemon
2022-08-15T10:45:17.291079441Z level=info msg="  --config=''" subsys=daemon
2022-08-15T10:45:17.291081308Z level=info msg="  --config-dir='/tmp/cilium/config-map'" subsys=daemon
2022-08-15T10:45:17.291083630Z level=info msg="  --conntrack-gc-interval='0s'" subsys=daemon
2022-08-15T10:45:17.291085573Z level=info msg="  --crd-wait-timeout='5m0s'" subsys=daemon
2022-08-15T10:45:17.291087496Z level=info msg="  --datapath-mode='veth'" subsys=daemon
2022-08-15T10:45:17.291089380Z level=info msg="  --debug='false'" subsys=daemon
2022-08-15T10:45:17.291091378Z level=info msg="  --debug-verbose=''" subsys=daemon
2022-08-15T10:45:17.291093275Z level=info msg="  --derive-masquerade-ip-addr-from-device=''" subsys=daemon
2022-08-15T10:45:17.291095302Z level=info msg="  --devices=''" subsys=daemon
2022-08-15T10:45:17.291097192Z level=info msg="  --direct-routing-device=''" subsys=daemon
2022-08-15T10:45:17.291099223Z level=info msg="  --disable-cnp-status-updates='true'" subsys=daemon
2022-08-15T10:45:17.291101136Z level=info msg="  --disable-conntrack='false'" subsys=daemon
2022-08-15T10:45:17.291103010Z level=info msg="  --disable-endpoint-crd='false'" subsys=daemon
2022-08-15T10:45:17.291104934Z level=info msg="  --disable-envoy-version-check='false'" subsys=daemon
2022-08-15T10:45:17.291106874Z level=info msg="  --disable-iptables-feeder-rules=''" subsys=daemon
2022-08-15T10:45:17.291108786Z level=info msg="  --dns-max-ips-per-restored-rule='1000'" subsys=daemon
2022-08-15T10:45:17.291110706Z level=info msg="  --dns-policy-unload-on-shutdown='false'" subsys=daemon
2022-08-15T10:45:17.291112729Z level=info msg="  --dnsproxy-concurrency-limit='0'" subsys=daemon
2022-08-15T10:45:17.291116434Z level=info msg="  --dnsproxy-concurrency-processing-grace-period='0s'" subsys=daemon
2022-08-15T10:45:17.291119419Z level=info msg="  --egress-masquerade-interfaces='eth0'" subsys=daemon
2022-08-15T10:45:17.291122553Z level=info msg="  --egress-multi-home-ip-rule-compat='false'" subsys=daemon
2022-08-15T10:45:17.291125652Z level=info msg="  --enable-auto-protect-node-port-range='true'" subsys=daemon
2022-08-15T10:45:17.291131361Z level=info msg="  --enable-bandwidth-manager='false'" subsys=daemon
2022-08-15T10:45:17.291133433Z level=info msg="  --enable-bbr='false'" subsys=daemon
2022-08-15T10:45:17.291135313Z level=info msg="  --enable-bgp-control-plane='false'" subsys=daemon
2022-08-15T10:45:17.291137211Z level=info msg="  --enable-bpf-clock-probe='true'" subsys=daemon
2022-08-15T10:45:17.291139134Z level=info msg="  --enable-bpf-masquerade='false'" subsys=daemon
2022-08-15T10:45:17.291141017Z level=info msg="  --enable-bpf-tproxy='false'" subsys=daemon
2022-08-15T10:45:17.291142901Z level=info msg="  --enable-cilium-endpoint-slice='false'" subsys=daemon
2022-08-15T10:45:17.291144821Z level=info msg="  --enable-custom-calls='false'" subsys=daemon
2022-08-15T10:45:17.291146723Z level=info msg="  --enable-endpoint-health-checking='true'" subsys=daemon
2022-08-15T10:45:17.291148650Z level=info msg="  --enable-endpoint-routes='true'" subsys=daemon
2022-08-15T10:45:17.291150557Z level=info msg="  --enable-envoy-config='false'" subsys=daemon
2022-08-15T10:45:17.291152880Z level=info msg="  --enable-external-ips='true'" subsys=daemon
2022-08-15T10:45:17.291154817Z level=info msg="  --enable-health-check-nodeport='true'" subsys=daemon
2022-08-15T10:45:17.291156757Z level=info msg="  --enable-health-checking='true'" subsys=daemon
2022-08-15T10:45:17.291158653Z level=info msg="  --enable-host-firewall='false'" subsys=daemon
2022-08-15T10:45:17.291160563Z level=info msg="  --enable-host-legacy-routing='false'" subsys=daemon
2022-08-15T10:45:17.291162493Z level=info msg="  --enable-host-port='true'" subsys=daemon
2022-08-15T10:45:17.291164406Z level=info msg="  --enable-host-reachable-services='false'" subsys=daemon
2022-08-15T10:45:17.291166301Z level=info msg="  --enable-hubble='true'" subsys=daemon
2022-08-15T10:45:17.291168189Z level=info msg="  --enable-hubble-recorder-api='true'" subsys=daemon
2022-08-15T10:45:17.291170106Z level=info msg="  --enable-icmp-rules='true'" subsys=daemon
2022-08-15T10:45:17.291171993Z level=info msg="  --enable-identity-mark='true'" subsys=daemon
2022-08-15T10:45:17.291173948Z level=info msg="  --enable-ip-masq-agent='false'" subsys=daemon
2022-08-15T10:45:17.291175924Z level=info msg="  --enable-ipsec='false'" subsys=daemon
2022-08-15T10:45:17.291177829Z level=info msg="  --enable-ipv4='true'" subsys=daemon
2022-08-15T10:45:17.291179734Z level=info msg="  --enable-ipv4-egress-gateway='false'" subsys=daemon
2022-08-15T10:45:17.291181653Z level=info msg="  --enable-ipv4-fragment-tracking='true'" subsys=daemon
2022-08-15T10:45:17.291183592Z level=info msg="  --enable-ipv4-masquerade='true'" subsys=daemon
2022-08-15T10:45:17.291185494Z level=info msg="  --enable-ipv6='false'" subsys=daemon
2022-08-15T10:45:17.291187392Z level=info msg="  --enable-ipv6-masquerade='true'" subsys=daemon
2022-08-15T10:45:17.291190985Z level=info msg="  --enable-ipv6-ndp='false'" subsys=daemon
2022-08-15T10:45:17.291192996Z level=info msg="  --enable-k8s-api-discovery='false'" subsys=daemon
2022-08-15T10:45:17.291196067Z level=info msg="  --enable-k8s-endpoint-slice='true'" subsys=daemon
2022-08-15T10:45:17.291199424Z level=info msg="  --enable-k8s-event-handover='false'" subsys=daemon
2022-08-15T10:45:17.291202706Z level=info msg="  --enable-k8s-terminating-endpoint='true'" subsys=daemon
2022-08-15T10:45:17.291205813Z level=info msg="  --enable-l2-neigh-discovery='true'" subsys=daemon
2022-08-15T10:45:17.291208741Z level=info msg="  --enable-l7-proxy='true'" subsys=daemon
2022-08-15T10:45:17.291211864Z level=info msg="  --enable-local-node-route='true'" subsys=daemon
2022-08-15T10:45:17.291214704Z level=info msg="  --enable-local-redirect-policy='false'" subsys=daemon
2022-08-15T10:45:17.291216692Z level=info msg="  --enable-mke='false'" subsys=daemon
2022-08-15T10:45:17.291218609Z level=info msg="  --enable-monitor='true'" subsys=daemon
2022-08-15T10:45:17.291220543Z level=info msg="  --enable-node-port='false'" subsys=daemon
2022-08-15T10:45:17.291222468Z level=info msg="  --enable-policy='default'" subsys=daemon
2022-08-15T10:45:17.291224660Z level=info msg="  --enable-recorder='false'" subsys=daemon
2022-08-15T10:45:17.291226600Z level=info msg="  --enable-remote-node-identity='true'" subsys=daemon
2022-08-15T10:45:17.291228569Z level=info msg="  --enable-runtime-device-detection='false'" subsys=daemon
2022-08-15T10:45:17.291230501Z level=info msg="  --enable-selective-regeneration='true'" subsys=daemon
2022-08-15T10:45:17.291232393Z level=info msg="  --enable-service-topology='false'" subsys=daemon
2022-08-15T10:45:17.291234305Z level=info msg="  --enable-session-affinity='false'" subsys=daemon
2022-08-15T10:45:17.291236193Z level=info msg="  --enable-svc-source-range-check='true'" subsys=daemon
2022-08-15T10:45:17.291238088Z level=info msg="  --enable-tracing='false'" subsys=daemon
2022-08-15T10:45:17.291239987Z level=info msg="  --enable-unreachable-routes='false'" subsys=daemon
2022-08-15T10:45:17.291241895Z level=info msg="  --enable-vtep='false'" subsys=daemon
2022-08-15T10:45:17.291243761Z level=info msg="  --enable-well-known-identities='false'" subsys=daemon
2022-08-15T10:45:17.291245692Z level=info msg="  --enable-wireguard='false'" subsys=daemon
2022-08-15T10:45:17.291247585Z level=info msg="  --enable-wireguard-userspace-fallback='false'" subsys=daemon
2022-08-15T10:45:17.291249522Z level=info msg="  --enable-xdp-prefilter='false'" subsys=daemon
2022-08-15T10:45:17.291251410Z level=info msg="  --enable-xt-socket-fallback='true'" subsys=daemon
2022-08-15T10:45:17.291253346Z level=info msg="  --encrypt-interface=''" subsys=daemon
2022-08-15T10:45:17.291255246Z level=info msg="  --encrypt-node='false'" subsys=daemon
2022-08-15T10:45:17.291258863Z level=info msg="  --endpoint-gc-interval='5m0s'" subsys=daemon
2022-08-15T10:45:17.291260846Z level=info msg="  --endpoint-interface-name-prefix=''" subsys=daemon
2022-08-15T10:45:17.291262790Z level=info msg="  --endpoint-queue-size='25'" subsys=daemon
2022-08-15T10:45:17.291264738Z level=info msg="  --endpoint-status=''" subsys=daemon
2022-08-15T10:45:17.291266635Z level=info msg="  --envoy-config-timeout='2m0s'" subsys=daemon
2022-08-15T10:45:17.291268527Z level=info msg="  --envoy-log=''" subsys=daemon
2022-08-15T10:45:17.291270411Z level=info msg="  --exclude-local-address=''" subsys=daemon
2022-08-15T10:45:17.291272299Z level=info msg="  --fixed-identity-mapping=''" subsys=daemon
2022-08-15T10:45:17.291274197Z level=info msg="  --force-local-policy-eval-at-source='true'" subsys=daemon
2022-08-15T10:45:17.291276288Z level=info msg="  --fqdn-regex-compile-lru-size='1024'" subsys=daemon
2022-08-15T10:45:17.291279966Z level=info msg="  --gops-port='9890'" subsys=daemon
2022-08-15T10:45:17.291282610Z level=info msg="  --host-reachable-services-protos='tcp,udp'" subsys=daemon
2022-08-15T10:45:17.291285964Z level=info msg="  --http-403-msg=''" subsys=daemon
2022-08-15T10:45:17.291288945Z level=info msg="  --http-idle-timeout='0'" subsys=daemon
2022-08-15T10:45:17.291291999Z level=info msg="  --http-max-grpc-timeout='0'" subsys=daemon
2022-08-15T10:45:17.291293986Z level=info msg="  --http-normalize-path='true'" subsys=daemon
2022-08-15T10:45:17.291296222Z level=info msg="  --http-request-timeout='3600'" subsys=daemon
2022-08-15T10:45:17.291298164Z level=info msg="  --http-retry-count='3'" subsys=daemon
2022-08-15T10:45:17.291300064Z level=info msg="  --http-retry-timeout='0'" subsys=daemon
2022-08-15T10:45:17.291301938Z level=info msg="  --hubble-disable-tls='false'" subsys=daemon
2022-08-15T10:45:17.291303873Z level=info msg="  --hubble-event-buffer-capacity='4095'" subsys=daemon
2022-08-15T10:45:17.291305811Z level=info msg="  --hubble-event-queue-size='0'" subsys=daemon
2022-08-15T10:45:17.291307731Z level=info msg="  --hubble-export-file-compress='false'" subsys=daemon
2022-08-15T10:45:17.291309658Z level=info msg="  --hubble-export-file-max-backups='5'" subsys=daemon
2022-08-15T10:45:17.291311565Z level=info msg="  --hubble-export-file-max-size-mb='10'" subsys=daemon
2022-08-15T10:45:17.291313465Z level=info msg="  --hubble-export-file-path=''" subsys=daemon
2022-08-15T10:45:17.291315367Z level=info msg="  --hubble-listen-address=':4244'" subsys=daemon
2022-08-15T10:45:17.291317257Z level=info msg="  --hubble-metrics=''" subsys=daemon
2022-08-15T10:45:17.291319176Z level=info msg="  --hubble-metrics-server=''" subsys=daemon
2022-08-15T10:45:17.291321092Z level=info msg="  --hubble-recorder-sink-queue-size='1024'" subsys=daemon
2022-08-15T10:45:17.291324646Z level=info msg="  --hubble-recorder-storage-path='/var/run/cilium/pcaps'" subsys=daemon
2022-08-15T10:45:17.291326646Z level=info msg="  --hubble-socket-path='/var/run/cilium/hubble.sock'" subsys=daemon
2022-08-15T10:45:17.291328570Z level=info msg="  --hubble-tls-cert-file='/var/lib/cilium/tls/hubble/server.crt'" subsys=daemon
2022-08-15T10:45:17.291330571Z level=info msg="  --hubble-tls-client-ca-files='/var/lib/cilium/tls/hubble/client-ca.crt'" subsys=daemon
2022-08-15T10:45:17.291332601Z level=info msg="  --hubble-tls-key-file='/var/lib/cilium/tls/hubble/server.key'" subsys=daemon
2022-08-15T10:45:17.291334546Z level=info msg="  --identity-allocation-mode='crd'" subsys=daemon
2022-08-15T10:45:17.291336450Z level=info msg="  --identity-change-grace-period='5s'" subsys=daemon
2022-08-15T10:45:17.291338400Z level=info msg="  --identity-restore-grace-period='10m0s'" subsys=daemon
2022-08-15T10:45:17.291340393Z level=info msg="  --install-egress-gateway-routes='false'" subsys=daemon
2022-08-15T10:45:17.291342303Z level=info msg="  --install-iptables-rules='true'" subsys=daemon
2022-08-15T10:45:17.291344179Z level=info msg="  --install-no-conntrack-iptables-rules='false'" subsys=daemon
2022-08-15T10:45:17.291346136Z level=info msg="  --ip-allocation-timeout='2m0s'" subsys=daemon
2022-08-15T10:45:17.291348026Z level=info msg="  --ip-masq-agent-config-path='/etc/config/ip-masq-agent'" subsys=daemon
2022-08-15T10:45:17.291349970Z level=info msg="  --ipam='eni'" subsys=daemon
2022-08-15T10:45:17.291351877Z level=info msg="  --ipsec-key-file=''" subsys=daemon
2022-08-15T10:45:17.291354008Z level=info msg="  --iptables-lock-timeout='5s'" subsys=daemon
2022-08-15T10:45:17.291357726Z level=info msg="  --iptables-random-fully='false'" subsys=daemon
2022-08-15T10:45:17.291360352Z level=info msg="  --ipv4-native-routing-cidr=''" subsys=daemon
2022-08-15T10:45:17.291363711Z level=info msg="  --ipv4-node='auto'" subsys=daemon
2022-08-15T10:45:17.291366880Z level=info msg="  --ipv4-pod-subnets=''" subsys=daemon
2022-08-15T10:45:17.291369783Z level=info msg="  --ipv4-range='auto'" subsys=daemon
2022-08-15T10:45:17.291371747Z level=info msg="  --ipv4-service-loopback-address='169.254.42.1'" subsys=daemon
2022-08-15T10:45:17.291373712Z level=info msg="  --ipv4-service-range='auto'" subsys=daemon
2022-08-15T10:45:17.291375588Z level=info msg="  --ipv6-cluster-alloc-cidr='f00d::/64'" subsys=daemon
2022-08-15T10:45:17.291377488Z level=info msg="  --ipv6-mcast-device=''" subsys=daemon
2022-08-15T10:45:17.291379376Z level=info msg="  --ipv6-native-routing-cidr=''" subsys=daemon
2022-08-15T10:45:17.291381257Z level=info msg="  --ipv6-node='auto'" subsys=daemon
2022-08-15T10:45:17.291383125Z level=info msg="  --ipv6-pod-subnets=''" subsys=daemon
2022-08-15T10:45:17.291384971Z level=info msg="  --ipv6-range='auto'" subsys=daemon
2022-08-15T10:45:17.291388524Z level=info msg="  --ipv6-service-range='auto'" subsys=daemon
2022-08-15T10:45:17.291390497Z level=info msg="  --ipvlan-master-device='undefined'" subsys=daemon
2022-08-15T10:45:17.291392390Z level=info msg="  --join-cluster='false'" subsys=daemon
2022-08-15T10:45:17.291394294Z level=info msg="  --k8s-api-server=''" subsys=daemon
2022-08-15T10:45:17.291396187Z level=info msg="  --k8s-heartbeat-timeout='30s'" subsys=daemon
2022-08-15T10:45:17.291398078Z level=info msg="  --k8s-kubeconfig-path=''" subsys=daemon
2022-08-15T10:45:17.291399966Z level=info msg="  --k8s-namespace='kube-system'" subsys=daemon
2022-08-15T10:45:17.291401888Z level=info msg="  --k8s-require-ipv4-pod-cidr='false'" subsys=daemon
2022-08-15T10:45:17.291403802Z level=info msg="  --k8s-require-ipv6-pod-cidr='false'" subsys=daemon
2022-08-15T10:45:17.291405691Z level=info msg="  --k8s-service-cache-size='128'" subsys=daemon
2022-08-15T10:45:17.291407674Z level=info msg="  --k8s-service-proxy-name=''" subsys=daemon
2022-08-15T10:45:17.291409582Z level=info msg="  --k8s-sync-timeout='3m0s'" subsys=daemon
2022-08-15T10:45:17.291411540Z level=info msg="  --k8s-watcher-endpoint-selector='metadata.name!=kube-scheduler,metadata.name!=kube-controller-manager,metadata.name!=etcd-operator,metadata.name!=gcp-controller-manager'" subsys=daemon
2022-08-15T10:45:17.291413726Z level=info msg="  --keep-config='false'" subsys=daemon
2022-08-15T10:45:17.291415608Z level=info msg="  --kube-proxy-replacement='disabled'" subsys=daemon
2022-08-15T10:45:17.291417523Z level=info msg="  --kube-proxy-replacement-healthz-bind-address=''" subsys=daemon
2022-08-15T10:45:17.291419471Z level=info msg="  --kvstore=''" subsys=daemon
2022-08-15T10:45:17.291421813Z level=info msg="  --kvstore-connectivity-timeout='2m0s'" subsys=daemon
2022-08-15T10:45:17.291423752Z level=info msg="  --kvstore-lease-ttl='15m0s'" subsys=daemon
2022-08-15T10:45:17.291425647Z level=info msg="  --kvstore-max-consecutive-quorum-errors='2'" subsys=daemon
2022-08-15T10:45:17.291427576Z level=info msg="  --kvstore-opt=''" subsys=daemon
2022-08-15T10:45:17.291429450Z level=info msg="  --kvstore-periodic-sync='5m0s'" subsys=daemon
2022-08-15T10:45:17.291431347Z level=info msg="  --label-prefix-file=''" subsys=daemon
2022-08-15T10:45:17.291433891Z level=info msg="  --labels=''" subsys=daemon
2022-08-15T10:45:17.291437534Z level=info msg="  --lib-dir='/var/lib/cilium'" subsys=daemon
2022-08-15T10:45:17.291440631Z level=info msg="  --local-max-addr-scope='252'" subsys=daemon
2022-08-15T10:45:17.291443992Z level=info msg="  --local-router-ipv4=''" subsys=daemon
2022-08-15T10:45:17.291447260Z level=info msg="  --local-router-ipv6=''" subsys=daemon
2022-08-15T10:45:17.291449224Z level=info msg="  --log-driver=''" subsys=daemon
2022-08-15T10:45:17.291451083Z level=info msg="  --log-opt=''" subsys=daemon
2022-08-15T10:45:17.291454532Z level=info msg="  --log-system-load='false'" subsys=daemon
2022-08-15T10:45:17.291456526Z level=info msg="  --max-controller-interval='0'" subsys=daemon
2022-08-15T10:45:17.291458421Z level=info msg="  --metrics=''" subsys=daemon
2022-08-15T10:45:17.291460312Z level=info msg="  --mke-cgroup-mount=''" subsys=daemon
2022-08-15T10:45:17.291464157Z level=info msg="  --monitor-aggregation='none'" subsys=daemon
2022-08-15T10:45:17.291466131Z level=info msg="  --monitor-aggregation-flags='all'" subsys=daemon
2022-08-15T10:45:17.291468043Z level=info msg="  --monitor-aggregation-interval='5s'" subsys=daemon
2022-08-15T10:45:17.291469975Z level=info msg="  --monitor-queue-size='0'" subsys=daemon
2022-08-15T10:45:17.291471870Z level=info msg="  --mtu='0'" subsys=daemon
2022-08-15T10:45:17.291473753Z level=info msg="  --node-port-acceleration='disabled'" subsys=daemon
2022-08-15T10:45:17.291475651Z level=info msg="  --node-port-algorithm='random'" subsys=daemon
2022-08-15T10:45:17.291477540Z level=info msg="  --node-port-bind-protection='true'" subsys=daemon
2022-08-15T10:45:17.291479501Z level=info msg="  --node-port-mode='snat'" subsys=daemon
2022-08-15T10:45:17.291481390Z level=info msg="  --node-port-range='30000,32767'" subsys=daemon
2022-08-15T10:45:17.291483360Z level=info msg="  --policy-audit-mode='false'" subsys=daemon
2022-08-15T10:45:17.291485280Z level=info msg="  --policy-queue-size='100'" subsys=daemon
2022-08-15T10:45:17.291487177Z level=info msg="  --policy-trigger-interval='1s'" subsys=daemon
2022-08-15T10:45:17.291489075Z level=info msg="  --pprof='false'" subsys=daemon
2022-08-15T10:45:17.291490991Z level=info msg="  --pprof-port='6060'" subsys=daemon
2022-08-15T10:45:17.291492855Z level=info msg="  --preallocate-bpf-maps='false'" subsys=daemon
2022-08-15T10:45:17.291494755Z level=info msg="  --prepend-iptables-chains='true'" subsys=daemon
2022-08-15T10:45:17.291496911Z level=info msg="  --procfs='/host/proc'" subsys=daemon
2022-08-15T10:45:17.291498825Z level=info msg="  --prometheus-serve-addr=':9962'" subsys=daemon
2022-08-15T10:45:17.291500729Z level=info msg="  --proxy-connect-timeout='1'" subsys=daemon
2022-08-15T10:45:17.291502624Z level=info msg="  --proxy-gid='1337'" subsys=daemon
2022-08-15T10:45:17.291504506Z level=info msg="  --proxy-max-connection-duration-seconds='0'" subsys=daemon
2022-08-15T10:45:17.291506448Z level=info msg="  --proxy-max-requests-per-connection='0'" subsys=daemon
2022-08-15T10:45:17.291508376Z level=info msg="  --proxy-prometheus-port='0'" subsys=daemon
2022-08-15T10:45:17.291510839Z level=info msg="  --read-cni-conf=''" subsys=daemon
2022-08-15T10:45:17.291514894Z level=info msg="  --restore='true'" subsys=daemon
2022-08-15T10:45:17.291518028Z level=info msg="  --route-metric='0'" subsys=daemon
2022-08-15T10:45:17.291523927Z level=info msg="  --sidecar-istio-proxy-image='cilium/istio_proxy'" subsys=daemon
2022-08-15T10:45:17.291526358Z level=info msg="  --single-cluster-route='false'" subsys=daemon
2022-08-15T10:45:17.291528299Z level=info msg="  --socket-path='/var/run/cilium/cilium.sock'" subsys=daemon
2022-08-15T10:45:17.291530205Z level=info msg="  --sockops-enable='false'" subsys=daemon
2022-08-15T10:45:17.291532081Z level=info msg="  --state-dir='/var/run/cilium'" subsys=daemon
2022-08-15T10:45:17.291533984Z level=info msg="  --tofqdns-dns-reject-response-code='refused'" subsys=daemon
2022-08-15T10:45:17.291535935Z level=info msg="  --tofqdns-enable-dns-compression='true'" subsys=daemon
2022-08-15T10:45:17.291537853Z level=info msg="  --tofqdns-endpoint-max-ip-per-hostname='50'" subsys=daemon
2022-08-15T10:45:17.291539798Z level=info msg="  --tofqdns-idle-connection-grace-period='0s'" subsys=daemon
2022-08-15T10:45:17.291541717Z level=info msg="  --tofqdns-max-deferred-connection-deletes='10000'" subsys=daemon
2022-08-15T10:45:17.291543665Z level=info msg="  --tofqdns-min-ttl='3600'" subsys=daemon
2022-08-15T10:45:17.291545533Z level=info msg="  --tofqdns-pre-cache=''" subsys=daemon
2022-08-15T10:45:17.291547400Z level=info msg="  --tofqdns-proxy-port='0'" subsys=daemon
2022-08-15T10:45:17.291549299Z level=info msg="  --tofqdns-proxy-response-max-delay='100ms'" subsys=daemon
2022-08-15T10:45:17.291551194Z level=info msg="  --trace-payloadlen='128'" subsys=daemon
2022-08-15T10:45:17.291553068Z level=info msg="  --tunnel='disabled'" subsys=daemon
2022-08-15T10:45:17.291554934Z level=info msg="  --tunnel-port='0'" subsys=daemon
2022-08-15T10:45:17.291556806Z level=info msg="  --version='false'" subsys=daemon
2022-08-15T10:45:17.291558667Z level=info msg="  --vlan-bpf-bypass=''" subsys=daemon
2022-08-15T10:45:17.291560531Z level=info msg="  --vtep-cidr=''" subsys=daemon
2022-08-15T10:45:17.291562467Z level=info msg="  --vtep-endpoint=''" subsys=daemon
2022-08-15T10:45:17.291564335Z level=info msg="  --vtep-mac=''" subsys=daemon
2022-08-15T10:45:17.291566209Z level=info msg="  --vtep-mask=''" subsys=daemon
2022-08-15T10:45:17.291568363Z level=info msg="  --write-cni-conf-when-ready=''" subsys=daemon
2022-08-15T10:45:17.291570303Z level=info msg="     _ _ _" subsys=daemon
2022-08-15T10:45:17.291572167Z level=info msg=" ___|_| |_|_ _ _____" subsys=daemon
2022-08-15T10:45:17.291574039Z level=info msg="|  _| | | | | |     |" subsys=daemon
2022-08-15T10:45:17.291576102Z level=info msg="|___|_|_|_|___|_|_|_|" subsys=daemon
2022-08-15T10:45:17.291578045Z level=info msg="Cilium 1.12.0 9447cd1 2022-07-19T12:22:00+02:00 go version go1.18.4 linux/amd64" subsys=daemon
2022-08-15T10:45:17.334066320Z level=info msg="cilium-envoy  version: e90612180b82d07c124bbf8e1ffe94a8d603f8ae/1.21.3/Distribution/RELEASE/BoringSSL" subsys=daemon
2022-08-15T10:45:17.355083161Z level=info msg="clang (10.0.0) and kernel (5.4.204) versions: OK!" subsys=linux-datapath
2022-08-15T10:45:17.359549270Z level=info msg="linking environment: OK!" subsys=linux-datapath
2022-08-15T10:45:17.504876060Z level=info msg="Detected mounted BPF filesystem at /sys/fs/bpf" subsys=bpf
2022-08-15T10:45:17.505203385Z level=info msg="Mounted cgroupv2 filesystem at /run/cilium/cgroupv2" subsys=cgroups
2022-08-15T10:45:17.505212207Z level=info msg="Parsing base label prefixes from default label list" subsys=labels-filter
2022-08-15T10:45:17.505366606Z level=info msg="Parsing additional label prefixes from user inputs: []" subsys=labels-filter
2022-08-15T10:45:17.505373531Z level=info msg="Final label prefixes to be used for identity evaluation:" subsys=labels-filter
2022-08-15T10:45:17.505417617Z level=info msg=" - reserved:.*" subsys=labels-filter
2022-08-15T10:45:17.505423327Z level=info msg=" - :io\\.kubernetes\\.pod\\.namespace" subsys=labels-filter
2022-08-15T10:45:17.505426336Z level=info msg=" - :io\\.cilium\\.k8s\\.namespace\\.labels" subsys=labels-filter
2022-08-15T10:45:17.505428883Z level=info msg=" - :app\\.kubernetes\\.io" subsys=labels-filter
2022-08-15T10:45:17.505431066Z level=info msg=" - !:io\\.kubernetes" subsys=labels-filter
2022-08-15T10:45:17.505482670Z level=info msg=" - !:kubernetes\\.io" subsys=labels-filter
2022-08-15T10:45:17.505527371Z level=info msg=" - !:.*beta\\.kubernetes\\.io" subsys=labels-filter
2022-08-15T10:45:17.505533232Z level=info msg=" - !:k8s\\.io" subsys=labels-filter
2022-08-15T10:45:17.505536578Z level=info msg=" - !:pod-template-generation" subsys=labels-filter
2022-08-15T10:45:17.505538935Z level=info msg=" - !:pod-template-hash" subsys=labels-filter
2022-08-15T10:45:17.505541068Z level=info msg=" - !:controller-revision-hash" subsys=labels-filter
2022-08-15T10:45:17.505543269Z level=info msg=" - !:annotation.*" subsys=labels-filter
2022-08-15T10:45:17.505545204Z level=info msg=" - !:etcd_node" subsys=labels-filter
2022-08-15T10:45:17.506317576Z level=info msg="Auto-disabling \"enable-bpf-clock-probe\" feature since KERNEL_HZ cannot be determined" error="Cannot probe CONFIG_HZ" subsys=daemon
2022-08-15T10:45:17.506494314Z level=info msg="Using autogenerated IPv4 allocation range" subsys=node v4Prefix=10.168.0.0/16
2022-08-15T10:45:17.506538602Z level=info msg="Initializing daemon" subsys=daemon
2022-08-15T10:45:17.528342963Z level=info msg="Establishing connection to apiserver" host="https://10.100.0.1:443" subsys=k8s
2022-08-15T10:45:17.560412936Z level=info msg="Connected to apiserver" subsys=k8s
2022-08-15T10:45:17.562379258Z level=info msg="Auto-disabling \"enable-node-port\", \"enable-external-ips\", \"bpf-lb-sock\", \"enable-host-port\" features and falling back to \"enable-host-legacy-routing\"" subsys=daemon
2022-08-15T10:45:17.562981410Z level=info msg="Inheriting MTU from external network interface" device=eth1 ipAddr=192.168.89.168 mtu=9001 subsys=mtu
2022-08-15T10:45:17.614828434Z level=info msg="Restoring 2 old CIDR identities" subsys=daemon
2022-08-15T10:45:17.615184549Z level=info msg="regenerating all endpoints" reason="one or more identities created or deleted" subsys=endpoint-manager
2022-08-15T10:45:17.615420961Z level=info msg="Envoy: Starting xDS gRPC server listening on /var/run/cilium/xds.sock" subsys=envoy-manager
2022-08-15T10:45:17.616604535Z level=info msg="Restored backends from maps" failedBackends=0 restoredBackends=8 subsys=service
2022-08-15T10:45:17.617038190Z level=info msg="Restored services from maps" failedServices=0 restoredServices=6 subsys=service
2022-08-15T10:45:17.617091887Z level=info msg="Reading old endpoints..." subsys=daemon
2022-08-15T10:45:17.617183858Z level=warning msg="Found incomplete restore directory /var/run/cilium/state/355_next. Removing it..." endpointID=355_next subsys=endpoint
2022-08-15T10:45:17.617190040Z level=warning msg="Found incomplete restore directory /var/run/cilium/state/1463_next. Removing it..." endpointID=1463_next subsys=endpoint
2022-08-15T10:45:17.617226550Z level=warning msg="Found incomplete restore directory /var/run/cilium/state/174_next. Removing it..." endpointID=174_next subsys=endpoint
2022-08-15T10:45:17.620969588Z level=info msg="Reusing previous DNS proxy port: 37765" subsys=daemon
2022-08-15T10:45:17.621024902Z level=info msg="Waiting until all Cilium CRDs are available" subsys=k8s
2022-08-15T10:45:18.621313919Z level=info msg="All Cilium CRDs have been found and are available" subsys=k8s
2022-08-15T10:45:18.722870635Z level=info msg="Retrieved node information from kubernetes node" nodeName=ip-192-168-92-209.ap-southeast-2.compute.internal subsys=k8s
2022-08-15T10:45:18.722951952Z level=info msg="Received own node information from API server" ipAddr.ipv4=192.168.92.209 ipAddr.ipv6="<nil>" k8sNodeIP=192.168.92.209 labels="map[alpha.eksctl.io/cluster-name:tammach alpha.eksctl.io/nodegroup-name:ng-44bae905 beta.kubernetes.io/arch:amd64 beta.kubernetes.io/instance-type:m5.large beta.kubernetes.io/os:linux eks.amazonaws.com/capacityType:ON_DEMAND eks.amazonaws.com/nodegroup:ng-44bae905 eks.amazonaws.com/nodegroup-image:ami-0af7b72d0747a471a eks.amazonaws.com/sourceLaunchTemplateId:lt-0a6969850f4efb72c eks.amazonaws.com/sourceLaunchTemplateVersion:1 failure-domain.beta.kubernetes.io/region:ap-southeast-2 failure-domain.beta.kubernetes.io/zone:ap-southeast-2b k8s.io/cloud-provider-aws:7d4df7d25eef3443f62456cdb5aa3f06 kubernetes.io/arch:amd64 kubernetes.io/hostname:ip-192-168-92-209.ap-southeast-2.compute.internal kubernetes.io/os:linux node.kubernetes.io/instance-type:m5.large topology.kubernetes.io/region:ap-southeast-2 topology.kubernetes.io/zone:ap-southeast-2b]" nodeName=ip-192-168-92-209.ap-southeast-2.compute.internal subsys=k8s v4Prefix="<nil>" v6Prefix="<nil>"
2022-08-15T10:45:18.722963405Z level=info msg="k8s mode: Allowing localhost to reach local endpoints" subsys=daemon
2022-08-15T10:45:18.723134343Z level=info msg="Detected devices" devices="[]" subsys=linux-datapath
2022-08-15T10:45:18.723141494Z level=info msg="Enabling k8s event listener" subsys=k8s-watcher
2022-08-15T10:45:18.728983507Z level=info msg="Removing stale endpoint interfaces" subsys=daemon
2022-08-15T10:45:18.729054133Z level=info msg="Skipping kvstore configuration" subsys=daemon
2022-08-15T10:45:18.729060797Z level=info msg="Waiting until local node addressing before starting watchers depending on it" subsys=k8s-watcher
2022-08-15T10:45:18.729075676Z level=info msg="Restored router address from node_config" file=/var/run/cilium/state/globals/node_config.h ipv4=192.168.92.230 ipv6="<nil>" subsys=node
2022-08-15T10:45:18.729080065Z level=info msg="Initializing node addressing" subsys=daemon
2022-08-15T10:45:18.729083017Z level=info msg="Initializing CRD-based IPAM" subsys=ipam
2022-08-15T10:45:18.729085662Z level=info msg="Subscribed to CiliumNode custom resource" name=ip-192-168-92-209.ap-southeast-2.compute.internal subsys=ipam
2022-08-15T10:45:18.729088772Z level=info msg="Creating or updating CiliumNode resource" node=ip-192-168-92-209.ap-southeast-2.compute.internal subsys=nodediscovery
2022-08-15T10:45:18.731621027Z level=info msg="Policy Add Request" ciliumNetworkPolicy="[&{EndpointSelector:{\"matchLabels\":{\"any:kind\":\"client\",\"k8s:io.kubernetes.pod.namespace\":\"cilium-test\"}} NodeSelector:{} Ingress:[] IngressDeny:[] Egress:[] EgressDeny:[{EgressCommonRule:{ToEndpoints:[{\"matchLabels\":{\"any:io.kubernetes.pod.namespace\":\"cilium-test\",\"any:kind\":\"echo\"}}] ToRequires:[] ToCIDR: ToCIDRSet:[] ToEntities:[] ToServices:[] ToGroups:[] aggregatedSelectors:[]} ToPorts:[{Ports:[{Port:http-8080 Protocol:TCP}]}] ICMPs:[]}] Labels:[k8s:io.cilium.k8s.policy.derived-from=CiliumNetworkPolicy k8s:io.cilium.k8s.policy.name=client-egress-to-echo-deny-named-port k8s:io.cilium.k8s.policy.namespace=cilium-test k8s:io.cilium.k8s.policy.uid=d9b44a21-fa5c-405d-a6f8-c673915dd483] Description:}]" policyAddRequest=49958590-e41f-4af7-808b-00bcc20597b7 subsys=daemon
2022-08-15T10:45:18.731639922Z level=info msg="Policy imported via API, recalculating..." policyAddRequest=49958590-e41f-4af7-808b-00bcc20597b7 policyRevision=2 subsys=daemon
2022-08-15T10:45:18.731704552Z level=info msg="Imported CiliumNetworkPolicy" ciliumNetworkPolicyName=client-egress-to-echo-deny-named-port k8sApiVersion=cilium.io/v2 k8sNamespace=cilium-test subsys=k8s-watcher
2022-08-15T10:45:18.772898057Z level=info msg="Waiting for CiliumNode custom resource to become available..." name=ip-192-168-92-209.ap-southeast-2.compute.internal subsys=ipam
2022-08-15T10:45:18.873877329Z level=info msg="Successfully synchronized CiliumNode custom resource" name=ip-192-168-92-209.ap-southeast-2.compute.internal subsys=ipam
2022-08-15T10:45:18.873912256Z level=info msg="Using autodetected primary VPC CIDR." subsys=ipam vpc-cidr=192.168.0.0/16
2022-08-15T10:45:18.873916078Z level=info msg="All required IPs are available in CRD-backed allocation pool" available=18 name=ip-192-168-92-209.ap-southeast-2.compute.internal required=8 subsys=ipam
2022-08-15T10:45:18.875372099Z level=info msg="Restoring endpoints..." subsys=daemon
2022-08-15T10:45:18.875389864Z level=info msg="Endpoints restored" failed=0 restored=6 subsys=daemon
2022-08-15T10:45:18.875393336Z level=info msg="Addressing information:" subsys=daemon
2022-08-15T10:45:18.875396346Z level=info msg="  Cluster-Name: tammach-ap-southeast-2-eksctl-io" subsys=daemon
2022-08-15T10:45:18.875399061Z level=info msg="  Cluster-ID: 0" subsys=daemon
2022-08-15T10:45:18.875412618Z level=info msg="  Local node-name: ip-192-168-92-209.ap-southeast-2.compute.internal" subsys=daemon
2022-08-15T10:45:18.875415780Z level=info msg="  Node-IPv6: <nil>" subsys=daemon
2022-08-15T10:45:18.875419438Z level=info msg="  External-Node IPv4: 192.168.92.209" subsys=daemon
2022-08-15T10:45:18.875422198Z level=info msg="  Internal-Node IPv4: 192.168.92.230" subsys=daemon
2022-08-15T10:45:18.875426114Z level=info msg="  IPv4 allocation prefix: 10.168.0.0/16" subsys=daemon
2022-08-15T10:45:18.875429407Z level=info msg="  IPv4 native routing prefix: 192.168.0.0/16" subsys=daemon
2022-08-15T10:45:18.875432419Z level=info msg="  Loopback IPv4: 169.254.42.1" subsys=daemon
2022-08-15T10:45:18.875435393Z level=info msg="  Local IPv4 addresses:" subsys=daemon
2022-08-15T10:45:18.875438417Z level=info msg="  - 192.168.92.209" subsys=daemon
2022-08-15T10:45:18.875441297Z level=info msg="  - 192.168.89.168" subsys=daemon
2022-08-15T10:45:18.875444304Z level=info msg="  - 192.168.92.230" subsys=daemon
2022-08-15T10:45:18.875448988Z level=info msg="Adding local node to cluster" node="{ip-192-168-92-209.ap-southeast-2.compute.internal tammach-ap-southeast-2-eksctl-io [{ExternalIP 3.106.134.192} {InternalIP 192.168.92.209} {CiliumInternalIP 192.168.92.230}] 10.168.0.0/16 [] <nil> [] 192.168.88.7 <nil> <nil> <nil> 0 local 0 map[alpha.eksctl.io/cluster-name:tammach alpha.eksctl.io/nodegroup-name:ng-44bae905 beta.kubernetes.io/arch:amd64 beta.kubernetes.io/instance-type:m5.large beta.kubernetes.io/os:linux eks.amazonaws.com/capacityType:ON_DEMAND eks.amazonaws.com/nodegroup:ng-44bae905 eks.amazonaws.com/nodegroup-image:ami-0af7b72d0747a471a eks.amazonaws.com/sourceLaunchTemplateId:lt-0a6969850f4efb72c eks.amazonaws.com/sourceLaunchTemplateVersion:1 failure-domain.beta.kubernetes.io/region:ap-southeast-2 failure-domain.beta.kubernetes.io/zone:ap-southeast-2b k8s.io/cloud-provider-aws:7d4df7d25eef3443f62456cdb5aa3f06 kubernetes.io/arch:amd64 kubernetes.io/hostname:ip-192-168-92-209.ap-southeast-2.compute.internal kubernetes.io/os:linux node.kubernetes.io/instance-type:m5.large topology.kubernetes.io/region:ap-southeast-2 topology.kubernetes.io/zone:ap-southeast-2b] 1 }" subsys=nodediscovery
2022-08-15T10:45:18.877172694Z level=info msg="Creating or updating CiliumNode resource" node=ip-192-168-92-209.ap-southeast-2.compute.internal subsys=nodediscovery
2022-08-15T10:45:18.877322162Z level=info msg="Waiting until all pre-existing resources have been received" subsys=k8s-watcher
2022-08-15T10:45:18.917316074Z level=info msg="Initializing identity allocator" subsys=identity-cache
2022-08-15T10:45:18.917338988Z level=info msg="Cluster-ID is not specified, skipping ClusterMesh initialization" subsys=daemon
2022-08-15T10:45:18.922532129Z level=info msg="Setting up BPF datapath" bpfClockSource=ktime bpfInsnSet=v2 subsys=datapath-loader
2022-08-15T10:45:18.922960054Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.core.bpf_jit_enable sysParamValue=1
2022-08-15T10:45:18.923024836Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.all.rp_filter sysParamValue=0
2022-08-15T10:45:18.923031398Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.fib_multipath_use_neigh sysParamValue=1
2022-08-15T10:45:18.923091795Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=kernel.unprivileged_bpf_disabled sysParamValue=1
2022-08-15T10:45:18.923096670Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=kernel.timer_migration sysParamValue=0
2022-08-15T10:45:18.923100068Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.eth0.rp_filter sysParamValue=2
2022-08-15T10:45:18.923216478Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.forwarding sysParamValue=1
2022-08-15T10:45:18.923256846Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.rp_filter sysParamValue=0
2022-08-15T10:45:18.923261672Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.accept_local sysParamValue=1
2022-08-15T10:45:18.923319266Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_host.send_redirects sysParamValue=0
2022-08-15T10:45:18.923411337Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.forwarding sysParamValue=1
2022-08-15T10:45:18.923417202Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.rp_filter sysParamValue=0
2022-08-15T10:45:18.923465375Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.accept_local sysParamValue=1
2022-08-15T10:45:18.923470479Z level=info msg="Setting sysctl" subsys=sysctl sysParamName=net.ipv4.conf.cilium_net.send_redirects sysParamValue=0
2022-08-15T10:45:18.932501960Z level=info msg="regenerating all endpoints" reason="one or more identities created or deleted" subsys=endpoint-manager
2022-08-15T10:45:19.348799757Z level=info msg="Iptables rules installed" subsys=iptables
2022-08-15T10:45:19.348821047Z level=info msg="Adding new proxy port rules for cilium-dns-egress:37765" proxy port name=cilium-dns-egress subsys=proxy
2022-08-15T10:45:19.350919410Z level=info msg="Iptables proxy rules installed" subsys=iptables
2022-08-15T10:45:19.351031393Z level=info msg="Serving cilium node monitor v1.2 API at unix:///var/run/cilium/monitor1_2.sock" subsys=monitor-agent
2022-08-15T10:45:19.351645964Z level=info msg="Validating configured node address ranges" subsys=daemon
2022-08-15T10:45:19.351655536Z level=info msg="Starting connection tracking garbage collector" subsys=daemon
2022-08-15T10:45:19.351751170Z level=info msg="Starting IP identity watcher" subsys=ipcache
2022-08-15T10:45:19.352718131Z level=info msg="Initial scan of connection tracking completed" subsys=ct-gc
2022-08-15T10:45:19.352821133Z level=info msg="Regenerating restored endpoints" numRestored=6 subsys=daemon
2022-08-15T10:45:19.352948998Z level=info msg="Conntrack garbage collector interval recalculated" deleteRatio=0.0007171630859375 newInterval=7m30s subsys=map-ct
2022-08-15T10:45:19.352996867Z level=info msg="Datapath signal listener running" subsys=signal
2022-08-15T10:45:19.353319124Z level=info msg="New endpoint" containerID=b3578b7cd0 datapathPolicyRevision=0 desiredPolicyRevision=0 endpointID=1463 identity=19010 ipv4=192.168.75.206 ipv6= k8sPodName=kube-system/coredns-68f7974869-jr5pj subsys=endpoint
2022-08-15T10:45:19.353429637Z level=info msg="New endpoint" containerID=3e73eedf69 datapathPolicyRevision=0 desiredPolicyRevision=0 endpointID=461 identity=25560 ipv4=192.168.84.15 ipv6= k8sPodName=cilium-test/client-6488dcf5d4-2kp5g subsys=endpoint
2022-08-15T10:45:19.353436105Z level=info msg="New endpoint" containerID=452406b908 datapathPolicyRevision=0 desiredPolicyRevision=0 endpointID=355 identity=61863 ipv4=192.168.71.72 ipv6= k8sPodName=cilium-test/echo-same-node-bc988d658-5x9cg subsys=endpoint
2022-08-15T10:45:19.353466628Z level=info msg="New endpoint" containerID=04b52c60ee datapathPolicyRevision=0 desiredPolicyRevision=0 endpointID=821 identity=17239 ipv4=192.168.72.160 ipv6= k8sPodName=cilium-test/client2-5998d566b4-7tg4f subsys=endpoint
2022-08-15T10:45:19.353519284Z level=info msg="New endpoint" containerID= datapathPolicyRevision=0 desiredPolicyRevision=0 endpointID=3357 identity=1 ipv4= ipv6= k8sPodName=/ subsys=endpoint
2022-08-15T10:45:19.353623193Z level=info msg="New endpoint" containerID=ae3f128d79 datapathPolicyRevision=0 desiredPolicyRevision=0 endpointID=174 identity=35525 ipv4=192.168.64.16 ipv6= k8sPodName=kube-system/hubble-ui-855687897c-2xjjg subsys=endpoint
2022-08-15T10:45:19.353628717Z level=info msg="Successfully restored endpoint. Scheduling regeneration" endpointID=174 subsys=daemon
2022-08-15T10:45:19.353631786Z level=info msg="Successfully restored endpoint. Scheduling regeneration" endpointID=3357 subsys=daemon
2022-08-15T10:45:19.353634783Z level=info msg="Successfully restored endpoint. Scheduling regeneration" endpointID=821 subsys=daemon
2022-08-15T10:45:19.353637664Z level=info msg="Successfully restored endpoint. Scheduling regeneration" endpointID=355 subsys=daemon
2022-08-15T10:45:19.353682280Z level=info msg="Successfully restored endpoint. Scheduling regeneration" endpointID=461 subsys=daemon
2022-08-15T10:45:19.353686461Z level=info msg="Successfully restored endpoint. Scheduling regeneration" endpointID=1463 subsys=daemon
2022-08-15T10:45:19.353907610Z level=info msg="Migration of ENI datapath successful, 0 endpoints were migrated and none failed." subsys=daemon
2022-08-15T10:45:19.353913553Z level=info msg="Launching Cilium health daemon" subsys=daemon
2022-08-15T10:45:19.364738212Z level=info msg="Launching Cilium health endpoint" subsys=daemon
2022-08-15T10:45:19.364814669Z level=info msg="Serving prometheus metrics on :9962" subsys=daemon
2022-08-15T10:45:19.364819769Z level=info msg="Started healthz status API server" address="127.0.0.1:9879" subsys=daemon
2022-08-15T10:45:19.374519281Z level=info msg="Reusing existing global key" key="k8s:io.cilium.k8s.namespace.labels.kubernetes.io/metadata.name=kube-system;k8s:io.cilium.k8s.policy.cluster=tammach-ap-southeast-2-eksctl-io;k8s:io.cilium.k8s.policy.serviceaccount=hubble-ui;k8s:io.kubernetes.pod.namespace=kube-system;k8s:k8s-app=hubble-ui;" subsys=allocator
2022-08-15T10:45:19.374926724Z level=info msg="Reusing existing global key" key="k8s:io.cilium.k8s.namespace.labels.kubernetes.io/metadata.name=cilium-test;k8s:io.cilium.k8s.policy.cluster=tammach-ap-southeast-2-eksctl-io;k8s:io.cilium.k8s.policy.serviceaccount=default;k8s:io.kubernetes.pod.namespace=cilium-test;k8s:kind=client;k8s:name=client2;k8s:other=client;" subsys=allocator
2022-08-15T10:45:19.375179269Z level=info msg="Reusing existing global key" key="k8s:io.cilium.k8s.namespace.labels.kubernetes.io/metadata.name=cilium-test;k8s:io.cilium.k8s.policy.cluster=tammach-ap-southeast-2-eksctl-io;k8s:io.cilium.k8s.policy.serviceaccount=default;k8s:io.kubernetes.pod.namespace=cilium-test;k8s:kind=echo;k8s:name=echo-same-node;k8s:other=echo;" subsys=allocator
2022-08-15T10:45:19.375413909Z level=info msg="Reusing existing global key" key="k8s:io.cilium.k8s.namespace.labels.kubernetes.io/metadata.name=cilium-test;k8s:io.cilium.k8s.policy.cluster=tammach-ap-southeast-2-eksctl-io;k8s:io.cilium.k8s.policy.serviceaccount=default;k8s:io.kubernetes.pod.namespace=cilium-test;k8s:kind=client;k8s:name=client;" subsys=allocator
2022-08-15T10:45:19.375658268Z level=info msg="Reusing existing global key" key="k8s:eks.amazonaws.com/component=coredns;k8s:io.cilium.k8s.namespace.labels.kubernetes.io/metadata.name=kube-system;k8s:io.cilium.k8s.policy.cluster=tammach-ap-southeast-2-eksctl-io;k8s:io.cilium.k8s.policy.serviceaccount=coredns;k8s:io.kubernetes.pod.namespace=kube-system;k8s:k8s-app=kube-dns;" subsys=allocator
2022-08-15T10:45:19.407589934Z panic: runtime error: invalid memory address or nil pointer dereference
2022-08-15T10:45:19.407616349Z [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x1df100f]
2022-08-15T10:45:19.407620789Z 
2022-08-15T10:45:19.407624311Z goroutine 827 [running]:
2022-08-15T10:45:19.407628024Z github.com/cilium/cilium/pkg/endpoint.(*Endpoint).GetNamedPortLocked(0xc00024a000, 0x50?, {0xc000ba1cd0, 0x9}, 0xc0?)
2022-08-15T10:45:19.407632118Z  /go/src/github.com/cilium/cilium/pkg/endpoint/policy.go:68 +0x4f
2022-08-15T10:45:19.407635961Z github.com/cilium/cilium/pkg/policy.(*L4Filter).ToMapState(0xc001080000, {0x3434870, 0xc00024a000}, 0x1)
2022-08-15T10:45:19.407639630Z  /go/src/github.com/cilium/cilium/pkg/policy/l4.go:345 +0x2c3
2022-08-15T10:45:19.407643550Z github.com/cilium/cilium/pkg/policy.(*EndpointPolicy).computeDirectionL4PolicyMapEntries(0xc0019e8540, 0xc0019e8540?, 0xc0019e80d8?, 0x40?)
2022-08-15T10:45:19.407646727Z  /go/src/github.com/cilium/cilium/pkg/policy/resolve.go:171 +0xba
2022-08-15T10:45:19.407649728Z github.com/cilium/cilium/pkg/policy.(*EndpointPolicy).computeDesiredL4PolicyMapEntries(0xc0019e8540)
2022-08-15T10:45:19.407652741Z  /go/src/github.com/cilium/cilium/pkg/policy/resolve.go:164 +0x56
2022-08-15T10:45:19.407655881Z github.com/cilium/cilium/pkg/policy.(*selectorPolicy).DistillPolicy(0xc000d180c0, {0x3434870?, 0xc00024a000}, 0x0)
2022-08-15T10:45:19.407659158Z  /go/src/github.com/cilium/cilium/pkg/policy/resolve.go:141 +0x105
2022-08-15T10:45:19.407661935Z github.com/cilium/cilium/pkg/policy.(*cachedSelectorPolicy).Consume(0xc000123770?, {0x3434870?, 0xc00024a000?})
2022-08-15T10:45:19.407664846Z  /go/src/github.com/cilium/cilium/pkg/policy/distillery.go:202 +0x35
2022-08-15T10:45:19.407667872Z github.com/cilium/cilium/pkg/endpoint.(*Endpoint).regeneratePolicy(0xc00024a000)
2022-08-15T10:45:19.407670950Z  /go/src/github.com/cilium/cilium/pkg/endpoint/policy.go:234 +0x3f7
2022-08-15T10:45:19.407755591Z github.com/cilium/cilium/pkg/endpoint.(*Endpoint).runPreCompilationSteps(0xc00024a000, 0xc000399400)
2022-08-15T10:45:19.407772085Z  /go/src/github.com/cilium/cilium/pkg/endpoint/bpf.go:814 +0x2c5
2022-08-15T10:45:19.407775725Z github.com/cilium/cilium/pkg/endpoint.(*Endpoint).regenerateBPF(0xc00024a000, 0xc000399400)
2022-08-15T10:45:19.407779954Z  /go/src/github.com/cilium/cilium/pkg/endpoint/bpf.go:584 +0x189
2022-08-15T10:45:19.407783892Z github.com/cilium/cilium/pkg/endpoint.(*Endpoint).regenerate(0xc00024a000, 0xc000399400)
2022-08-15T10:45:19.407787952Z  /go/src/github.com/cilium/cilium/pkg/endpoint/policy.go:398 +0x7a5
2022-08-15T10:45:19.407791030Z github.com/cilium/cilium/pkg/endpoint.(*EndpointRegenerationEvent).Handle(0xc0014a2b70, 0x29fef80?)
2022-08-15T10:45:19.407794237Z  /go/src/github.com/cilium/cilium/pkg/endpoint/events.go:53 +0x325
2022-08-15T10:45:19.407797174Z github.com/cilium/cilium/pkg/eventqueue.(*EventQueue).run.func1()
2022-08-15T10:45:19.407800206Z  /go/src/github.com/cilium/cilium/pkg/eventqueue/eventqueue.go:245 +0x13b
2022-08-15T10:45:19.407803292Z sync.(*Once).doSlow(0x22ef001?, 0x4422a5?)
2022-08-15T10:45:19.407806380Z  /usr/local/go/src/sync/once.go:68 +0xc2
2022-08-15T10:45:19.407809380Z sync.(*Once).Do(...)
2022-08-15T10:45:19.407812406Z  /usr/local/go/src/sync/once.go:59
2022-08-15T10:45:19.407815416Z github.com/cilium/cilium/pkg/eventqueue.(*EventQueue).run(0x0?)
2022-08-15T10:45:19.407818504Z  /go/src/github.com/cilium/cilium/pkg/eventqueue/eventqueue.go:233 +0x45
2022-08-15T10:45:19.407821483Z created by github.com/cilium/cilium/pkg/eventqueue.(*EventQueue).Run
2022-08-15T10:45:19.407825267Z  /go/src/github.com/cilium/cilium/pkg/eventqueue/eventqueue.go:229 +0x76
```

</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
